repl/src/main/scala/org/apache/spark/h2o/H2OLogging.scala

```diff
@@ -0,0 +1,94 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.apache.spark.h2o
+
+import org.apache.log4j.{Level, LogManager, PropertyConfigurator}
+import org.apache.spark.Logging
+import org.apache.spark.util.Utils
+import org.slf4j.impl.StaticLoggerBinder
+
+trait H2OLogging extends Logging {
```

Comment:
It seems like this class is based on https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/internal/Logging.scala. I would mention it in the code why it needs to be copy-pasted and also would mention the origin of the code, for the future tracking.